src/mococrw/openssl_wrap.h

```diff
@@ -441,6 +441,7 @@ enum class ASN1_NID : int {
     GivenName = NID_givenName,
     UserId = NID_userId,
     Title = NID_title,
+    Initials = NID_initials,  ///< initials NID enum entry
```

Comment:
I agree, especially since the other values are also undocumented. I raised this inconsistency, however I was explicitly asked to add the comments for the doxygen. It is also part of the acceptance criteria.